modules/bullet/collision_object_bullet.h

```diff
@@ -167,6 +167,7 @@ class CollisionObjectBullet : public RIDBullet {
 	_FORCE_INLINE_ const VSet<RID> &get_exceptions() const { return exceptions; }
 
 	_FORCE_INLINE_ void set_collision_layer(uint32_t p_layer) {
+		ERR_FAIL_COND_MSG(p_layer >= (1 << 20), "Collision layer must be smaller than " + itos(1 << 20) + ".");
```

Comment:
Why do you want to clamp to max 20 bits?
Also to me this check is useless since any 32 bit value is allowed here.

Reply:
![image](https://user-images.githubusercontent.com/8281454/72476265-247d5280-3830-11ea-9873-ccdef196ac82.png)
because the editor shows only 20 bits?

Reply:
This is not a good reason, because:
- In future we may preserve some channels for internal engine features.
- Why we have to limit to only 20 if we use 32 bit anyway?
- In past, we discussed to increase the number of slots since 20 is not enough for certain games; @kubecz3k I remember that we discussed about it.